FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuthErrors.h

```diff
@@ -429,6 +429,11 @@ typedef NS_ERROR_ENUM(FIRAuthErrorDomain, FIRAuthErrorCode){
         describing which step of the JWT parsing process failed.
      */
     FIRAuthErrorCodeMalformedJWT = 18000,
+
+    /** Raised when an Cloud Function returns a blocking error. Will include a message returned from
+     * the function.
+     */
+    FIRAuthErrorCodeBlockingCloudFunctionError = 17105,
```

Comment:
@renkelvin is this where you want L415 to be placed?

Reply:
Please place it right below `FIRAuthErrorCodeMissingOrInvalidNonce`, so that the errors are in order.

Reply:
Never mind. Just realized what you meant.